plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -763,6 +765,11 @@ else if (isTrinoView || isTrinoMaterializedView) {
             properties.put(AUTO_PURGE, true);
         }
 
+        String externalTablePurgeProperty = table.getParameters().get(EXTERNAL_TABLE_PURGE_KEY);
+        if (parseBoolean(externalTablePurgeProperty)) {
+            properties.put(EXTERNAL_TABLE_PURGE, true);
```

Comment:
Please disallow enabling this property on non-external tables and add a test.

Reply:
Made changes at the appropriate place[ inside createTable() ]